edit-post/api/index.js

```diff
@@ -1,4 +1,11 @@
+import { registerPlugin } from './plugin';
+import { PluginSidebarFill as PluginSidebar } from './components/plugin-sidebar';
```

Comment:
I'm quite confused about this.

In `plugin-sidebar/index.js`:

```js
export { PluginSidebarFill, PluginSidebarSlot as PluginSidebar };
```

In `edit-post/api/index.js`:

```js
import { PluginSidebarFill as PluginSidebar } from './components/plugin-sidebar';
```

What is "PluginSidebar" ? We're using it in two different ways.

Reply:
Further: Elsewhere we define slot as properties of the class. We should be consistent one way or the other.

See:

https://github.com/WordPress/gutenberg/blob/8a4d644a525c1463f469e525d5d31ad36f44e23c/components/popover/index.js#L332

Reply:
Because I didn't want ot expose the slot to the plugin developer, but it can't hurt I guess.